net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/Api/ServiceController.php

```diff
@@ -52,7 +53,20 @@ class ServiceController extends ApiMutableServiceControllerBase
     public function showconfAction()
     {
         $backend = new Backend();
-        $response = $backend->configdRun("wireguard showconf");
+        $response_org = $backend->configdRun("wireguard showconf");
+        $response = '';
+
+        $pubnames = $this->getPubkeyNames();
+        $rp_lines = preg_split('/\r\n|\r|\n/', $response_org);
```

Comment:
if you split and store you can override response and remove $response_org

Reply:
btw, shouldn't '\n' and/or explode() suffice?

Reply:
I have change the variables and move the function to the client model for reuse.